casanovo/config.py

```
@@ -26,6 +26,16 @@
 )
 
 
+def _int_or_float(value: Union[int, float]) -> Union[int, float]:
```

Ruff (0.16.1)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)